hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/tenantconfiguration/AuthenticationConfigurationView.java

```diff
@@ -48,24 +52,23 @@
 
     private final UiProperties uiProperties;
 
-    private final Binder<ProxySystemConfigWindow> binder;
-
-    AuthenticationConfigurationView(final VaadinMessageSource i18n, final UiProperties uiProperties,
-            final SecurityTokenGenerator securityTokenGenerator, final Binder<ProxySystemConfigWindow> binder) {
+    public AuthenticationConfigurationView(final VaadinMessageSource i18n, final UiProperties uiProperties,
+            final TenantConfigurationManagement tenantConfigurationManagement, final SystemManagement systemManagement,
+            final SecurityTokenGenerator securityTokenGenerator) {
+        super(tenantConfigurationManagement, systemManagement, securityTokenGenerator);
         this.i18n = i18n;
         this.targetSecurityTokenAuthenticationConfigurationItem = new TargetSecurityTokenAuthenticationConfigurationItem(
                 i18n);
-        this.certificateAuthenticationConfigurationItem = new CertificateAuthenticationConfigurationItem(i18n, binder);
+        this.certificateAuthenticationConfigurationItem = new CertificateAuthenticationConfigurationItem(i18n,
+                getBinder());
         this.gatewaySecurityTokenAuthenticationConfigurationItem = new GatewaySecurityTokenAuthenticationConfigurationItem(
-                i18n, securityTokenGenerator, binder);
+                i18n, securityTokenGenerator, getBinder());
         this.anonymousDownloadAuthenticationConfigurationItem = new AnonymousDownloadAuthenticationConfigurationItem(
                 i18n);
         this.uiProperties = uiProperties;
-        this.binder = binder;
-
-        init();
     }
 
+    @PostConstruct
```

Comment:
we can call init method from constructor. Just to have consistency unless there is a reason

Reply:
Some initialization has to be done after the parent's and child's constructor are executed. This initialization is implemented in the child class and should not be called in the parent's constructor. That's why `@PostConstruct` is needed.
Right now I replaced this with implementing `InitializingBean `and its `afterPropertiesSet()`.